### PR TITLE
MemoByAccount.tsx - build fix

### PR DESCRIPTION
**Pull request by mnaamani**

### CODE REVIEW

File: packages/joy-utils/src/memo/MemoView.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ class Component extends React.PureComponent<Props> {
       const md = u8aToString(memo).trim();
       if (preview) {
         const plainText = this.mdToPlainText(md);
-        const previewText = truncate(plainText, { length: 80, omission: '…' });
+        const previewText = truncate(plainText, { length: 80, omission: '…view full memo' });
```
Comment (siman) [inline packages/joy-utils/src/memo/MemoView.tsx]: I think it makes more sense to make  "view full memo" as a link (`<Link to=...>`) while other preview text as a plain text (span, not `Link` tag).

File: packages/app-addresses/src/MemoByAccount.tsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ type State = {
 
 export default class Component extends React.PureComponent<Props, State> {
 
-  static getDerivedStateFromProps (props: Props, currentState: State) {
+  static getDerivedStateFromProps (props: any, currentState: State): State | null {
```
Comment (siman) [inline packages/app-addresses/src/MemoByAccount.tsx]: Cool that you found the right place!  Could you try to change to this instead:
```suggestion
  static getDerivedStateFromProps (props: Props, currentState: State): State | null {
```

Comment (mnaamani) [reply packages/app-addresses/src/MemoByAccount.tsx]: Fixed. The Prop type needed to implement `RouteComponentProps`. I pushed a fix

File: packages/joy-utils/src/memo/MemoView.tsx (hunk 1 above)
Comment (mnaamani) [reply packages/joy-utils/src/memo/MemoView.tsx]: Good suggestion. Done.

**Pull request merged**